README.md

```diff
@@ -274,77 +274,77 @@ Lightning also adds a text column with all the hyperparameters for this experime
 
 ![tensorboard-support](docs/source/_static/images/tf_tags.png)
 
-## Lightning automates all of the following ([each is also configurable](https://williamfalcon.github.io/pytorch-lightning/Trainer/)):
+## Lightning automates all of the following ([each is also configurable](https://PyTorchLightning.github.io/pytorch-lightning/Trainer/)):
```

Comment:
all the following links shall be revisited according newly generated docs...

Reply:
quite a number of links are referring just a single line in docs so I would skip them and if needed just list the topic on the page... @williamFalcon ?

Reply:
What are you referring to exactly @Borda? Only one link per page? And not linking to each section fragment? 

If so, i'd agree with this. When joining this project I was pretty overwhelmed with how many links there were. I opened them all to start reading, then realized most pages are the same. 

Alternatively, if we were having links to all sections - I think the sections in the Readme should align with the sections in the docs. E.g. Training Loop should only have the Training Loop page links underneath.

Reply:
that was my thoughts, have links only to major section and alternatively add examples/notebooks where the particular applications are explained...